src/main/webapp/database-tables.js

```diff
@@ -68,6 +79,67 @@ function login() {
   });
 }
 
+//TODO: Pass another paramater to combine data conversion functions 
+function dataConversionMillis(column,tableData){
+    tableData.columnSchemas[column].schemaType = "TIMESTAMP";
+    for (var i =0; i<tableData.dataTable.length; i++) {
```

Comment:
Same comment (maybe index instead of i but not a big deal)

Reply:
Its more clear so I changed it!